google/cloud/spanner/keys.cc

```diff
@@ -14,12 +14,24 @@
 
 #include "google/cloud/spanner/keys.h"
 #include "google/cloud/spanner/value.h"
+#include <google/protobuf/util/message_differencer.h>
 #include <google/spanner/v1/keys.pb.h>
 
 namespace google {
 namespace cloud {
 namespace spanner {
 inline namespace SPANNER_CLIENT_NS {
+
+bool operator==(KeySet const& lhs, KeySet const& rhs) {
+  //    std::string delta;
+  google::protobuf::util::MessageDifferencer differencer;
+  //    differencer.ReportDifferencesToString(&delta);
+  auto const result = differencer.Compare(lhs.proto_, rhs.proto_);
+  return result;
```

Comment:
consider just `return differencer.Compare(...);`, since we don't need the intermediate `result` variable.

Reply:
Done.